### DISCUSSION TITLE
Documents eBPF/XDP

### DISCUSSION THREAD

**Discussion by SilvaMendes** [loxilb-io/loxilb#579] [General]
Good afternoon, I'm studying the loxilb project, and I would like to know if you have the development documentation for the ebpf modules for xdp applications?  Another thing, I made adjustments to the Dockerfile for testing and managed to reduce the image to 360mb and I would like to share it with you.  if it is of interest to you.  Sorry for the English, I'm using Google translator, I'm Brazilian and I'm not fluent.

**Comment by UltraInstinct14** [comment]
There are limited development documentation about ebpf modules. However you can feel free to discuss about it with loxilb slack community. And regarding the Dockerfile changes, you are welcome. Feel free to share the changes via a Pull Request. Maintainers. will be happy to review and accept it.

**Comment by TrekkieCoder** [comment]
Docker image has now reduced to ~230MB [link](https://github.com/loxilb-io/loxilb/issues/482) . Thanks !!!
